incubator/react-native-error-trace-decorator/src/extract_and_process_error_stack.ts

```diff
@@ -0,0 +1,92 @@
+import { info } from "@rnx-kit/console";
+import * as fse from "fs-extra";
+import * as ChildProcess from "child_process";
+
+interface IBundleInterface {
+  bundleIdentifier: string;
+  sourcemap: string;
+}
+
+interface IConfigFile {
+  configs: IBundleInterface[];
+}
+
+/**
+ *  Extracts and symbolicates error stack traces.
+ *
+ * @param errorFilePath   The path to the error file
+ * @param configFilePath  The path to the config file
+ */
+export const extractAndSymbolicateErrorStack = (
+  errorFilePath: string,
+  configFilePath: string
+) => {
+  // Read config file as an object
+  const configFile = JSON.parse(
+    fse.readFileSync(configFilePath, "utf-8")
```

Comment:
could we use the format parameterised or a constant? in line 32 its specified as "utf8"

Reply:
I don't think we need a constant for that.  "utf-8" denotes the encodingin which the file must be read.